docker-compose-no-secty-arm64.yml

```diff
@@ -107,7 +107,6 @@ services:
       ALL_SERVICES_REGISTRY_HOST: edgex-core-consul
       APP_SERVICES_CLIENTS_CORE_METADATA_HOST: edgex-core-metadata
       DEVICE_SERVICES_CLIENTS_CORE_METADATA_HOST: edgex-core-metadata
-      EDGEX_SECURITY_SECRET_STORE: "false"
```

Comment:
This break non-secure compose files. Not the fix we need.

Reply:
I'll try to add env_file: common.env and see if that fixes anything.